nominations/admin.py

```diff
@@ -166,8 +295,6 @@ def get_form(self, request, obj=None, **kwargs):
             'candidate_district',
             'candidate_city',
             'candidate_state',
-            'get_general_election',
-            'get_primary_election',
```

Comment:
Is this accidental? Looks like a messed up merge?

Reply:
It is accidental but it doesn't affect functionality because we're using fieldsets here now. Will just remove this part, thanks for flagging.